src/components/select-field/BaseSelectField.js

```diff
@@ -112,8 +114,6 @@ class BaseSelectField extends React.Component<Props, State> {
 
         this.selectFieldContainerRef = React.createRef();
 
-        this.searchInputRef = React.createRef();
```

Comment:
Removed because we already have this.searchInputRef being set later in the code.